contact-form.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link href="styles.css" rel="stylesheet">
+    <title>Contact Us</title>
+  </head>
+
+  <body>
+    <h1>Contact Us</h1>
+
+    <h2>Send Us Your Questions or Comments</h2>
+    <form>
+      Name:<br>
+      <input type="text" name="name"><br>
+      Email:<br>
+      <input type="text" name="email"><br><br>
+      <textarea rows="4" cols="50">
+      </textarea><br><br>
+      <input type="submit" value="Submit">
+    </form>
+
+  <script src="scripts.js"></script>
+  </body>
+
+
+</html>
```

Comment:
I had a chance to test this code and everything works great! The only issue I noticed is the missing line at the end of the closing html tag, though this is not critical to change (please see my comments on index.html file). Nice work Charles!